config/routes.rb

```diff
@@ -10,12 +15,22 @@
   get '/users/new', to: 'users#new', as: 'new_user'
   post '/users', to: 'users#create', as: 'users'
 
+
   put '/cart', to: "carts#update"
   delete '/cart', to: "carts#destroy"
   get '/cart', to: "carts#show"
   post '/cart', to: "carts#create"
 
+
+  resources :users, only: [:new, :create, :show, :edit, :update ]
+
   resources :items, only: [:index, :show]
   resources :categories, only: [ :index]
   get "/:category", to: "categories#show", as: "category"
+
+    namespace :admin do
+      get 'dashboard', to: "dashboard#index"
+      resources :items
+    end
+
```

Comment:
Watch the indents.. Also, we need to have /:category at the bottom. Otherwise we could get some errors